### PR TITLE
add fsck error workaround to contributing.md

### LINKED ISSUE #2887: repo fails git fsck
repro:

```console
$ git config --system receive.fsckObjects true
$ git clone git@github.com:jashkenas/underscore.git  # or `git submodule add -- git@github.com:jashkenas/underscore.git`
Cloning into 'underscore'...
remote: Enumerating objects: 215, done.
remote: Counting objects: 100% (215/215), done.
remote: Compressing objects: 100% (130/130), done.
error: object b252cd0cf660db8fea1a40ae7d3ba0e34a41ffd3: zeroPaddedFilemode: contains zero-padded file modes
fatal: fsck error in packed object
```

### PR DESCRIPTION
**Pull request by zackschuster**
fixes #2887

### CODE REVIEW

**Comment by ronnie32921981** [comment]
Stop sending me emails please

Ronald Steffens


On Apr 25, 2022, at 11:15, Zack Schuster ***@***.***> wrote:

﻿

fixes #2887<https://github.com/jashkenas/underscore/issues/2887>

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/jashkenas/underscore/pull/2956

Commit Summary

  *   46af674<https://github.com/jashkenas/underscore/pull/2956/commits/46af674b7fa25a21809a2c5ffd336d0b2d9d773e> add fsck error workaround to contributing.md #2887

File Changes

(1 file<https://github.com/jashkenas/underscore/pull/2956/files>)

  *   M CONTRIBUTING.md<https://github.com/jashkenas/underscore/pull/2956/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055> (3)

Patch Links:

  *   https://github.com/jashkenas/underscore/pull/2956.patch
  *   https://github.com/jashkenas/underscore/pull/2956.diff

**Comment by zackschuster** [comment]
github is sending you these emails; i'm merely contributing to the `jashkenas/underscore` repo. you'll need to update your subscription / notification settings in the website. apologies for the noise!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,3 +15,6 @@
 * Use the same coding style as the rest of the [codebase](https://github.com/jashkenas/underscore/blob/master/modules/index.js).
 
 * In your pull request, do not add documentation or re-build the minified `underscore-umd-min.js` file. We'll do those things before cutting a new release.
+
+## "Help, cloning fails with `fatal: fsck error in packed object`"
```
Comment (jgonggrijp) [inline CONTRIBUTING.md:19]: Maybe make this `###` to make clear that it is a subsection of the top-level heading.

(I wondered for a moment whether we should make the top-level heading `#` instead, but I think that is meant to rank below the overall README.)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,3 +15,6 @@
...
+This error is caused by zero-padded file modes in the commit history. As fixing this is highly destructive, we suggest ignoring these warnings. The simplest way is to instruct git to do so when cloning: `git clone --config transfer.fsckobjects=false git@github.com:jashkenas/underscore.git`
```
Comment (jgonggrijp) [inline CONTRIBUTING.md:20]: Maybe add a sentence to clarify that `jashkenas` can be replaced by your own username, depending on what you are cloning.

**Review by jgonggrijp — COMMENTED** [review]
Thanks! I have two minor nitpicks, but I'm happy with the text you proposed.

**Comment by jgonggrijp** [comment]
Thanks again!

**Pull request merged**